src/App.test.js

```diff
@@ -13,11 +15,41 @@ const messages = {
   en: messages_en,
 };
 
-it('renders welcome message', () => {
-  const { getByText } = render(
-    <IntlProvider locale="en" messages={messages['en']}>
-      <App />
-    </IntlProvider>,
-  );
-  expect(getByText('Learn React')).toBeInTheDocument();
+const response = {
+  data: {
+    user: {
+      Email: 'fcoronel@makingsense.com',
+    },
+  },
+};
+
+describe('Test App component', () => {
+  afterEach(cleanup);
```

Comment:
If I am not wrong, you are already doing it in line 3: `import 'react-testing-library/cleanup-after-each';`

If line 3 alone does not work, I think that we should remove that line.

Reply:
fixed and remove line 3

Reply:
Mmm, [Facebook](https://facebook.github.io/create-react-app/docs/getting-started) people are disappointing me. A lot of wonderful and magic things that they said in _create-react-app_ articles are not working.